array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,17 @@
+/* exported take */
+// create empty array and set a condition for task with empty array (array.length = 0);
+// run for loop but make i < count (instead of array.length), and push it to result array
+
+function take(array, count) {
+  var result = [];
+  if (array.length === 0) {
+    return result;
+  } else {
+    for (var i = 0; i < count; i++) {
```

Comment:
I think you need to worry about if `count` is larger than `array.length`, don't you?